README.md

```diff
@@ -1 +1,70 @@
-# webpack
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+ToDo App
+> A todo webapp configured with webpack
+
+
+## Built With
+
+- Major languages
+  -Java Script 
```

Comment:
- Please, kindly correct the spelling of "JavaScript" as supposed to "Java Script"